README.md

```diff
@@ -12,3 +12,22 @@ Here we document our workflows, styleguides and patterns.
 
 * [Frontend Styleguide](https://github.com/GeekHunter-Brasil/hello-world/tree/master/frontend-styleguide)
 * [Backend Styleguide](https://github.com/GeekHunter-Brasil/hello-world/tree/master/backend-styleguide)
+* [Development Styleguide](https://github.com/GeekHunter-Brasil/hello-world/tree/master/development-styleguide)
+
+
+# The Geek Manifest
+
+- No code is better than no code (if I didn't know how to program, how would I solve it?);
+- Delivered is when we can prove that the customer received value in production;
+  - Customer used;
+  - We collect usage behavior, we prove it works;
+  - We monitor bugs and performance issues to make sure it's done right;
+- TDD is an implicit requirement of any code intervention;
+- The sprint is always the guide, tasks outside of it should be the exception (“Top! Tell Vivi to prioritize for us!”)
```

Comment:
I think this quote could be "Top! Tell our PO to prioritize for us!"

Reply:
True! xD